deploy.yaml

```diff
@@ -21,7 +21,7 @@ spec:
       - env:
         - name: SERVE_HTTP
           value: "true"
-        image: perconalab/version-service:master-8ac1b21
+        image: perconalab/version-service:k8spxc-1071-06622d3
```

Comment:
If I understand correctly this file is only an example, so we should maybe just let it be like it was?

Reply:
reverted